lib/tasks/s3.rake

```diff
@@ -89,42 +88,6 @@ def ensure_s3_configured!
   end
 end
 
-task "s3:correct_acl" => :environment do
-  ensure_s3_configured!
-
-  if !SiteSetting.s3_use_acls
-    $stderr.puts "Not correcting ACLs as the site is configured to not use ACLs"
-    return
-  end
-
-  puts "ensuring public-read is set on every upload and optimized image"
-
-  i = 0
-
-  base_url = Discourse.store.absolute_base_url
-
-  objects = Upload.pluck(:id, :url).map { |array| array << :upload }
-  objects.concat(OptimizedImage.pluck(:id, :url).map { |array| array << :optimized_image })
-
-  puts "#{objects.length} objects found"
-
-  objects.each do |id, url, type|
-    i += 1
-    if !url.start_with?(base_url)
-      puts "Skipping #{type} #{id} since it is not stored on s3, url is #{url}"
-    else
-      begin
-        key = url[(base_url.length + 1)..-1]
-        object = Discourse.store.s3_helper.object(key)
-        object.acl.put(acl: "public-read")
-      rescue => e
-        puts "Skipping #{type} #{id} url is #{url} #{e}"
-      end
-    end
-    puts "#{i} done" if i % 100 == 0
-  end
-end
```

Comment:
This is unnecessary since we already have https://github.com/discourse/discourse/blob/ecc64e3925ed0bb868605c8652f41cb7fed3a6e3/lib/tasks/uploads.rake#L569 which does the same thing.